### PR TITLE
Update az cors add help doc

### LINKED ISSUE #5621: `storage cors add`: '--methods' unclear
Trying to add cors via api, documentation here is not clear... which format should I use for methods param on add? invested some time and not able to add cors via az client... methods param only accept one verb

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b2e3dfa7-e7ba-d8b9-9c8b-7a716016b7c0
* Version Independent ID: 249776f7-0d81-361b-822e-b87c2ca7fb23
* [Content](https://docs.microsoft.com/en-us/cli/azure/storage/cors?view=azure-cli-latest#az_storage_cors_list)
* [Content Source](https://github.com/Azure/azure-cli/blob/master/src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py)
* Service: storage

### PR DESCRIPTION
**Pull request by troydai**
Fixes #5621 

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](../doc/command_guidelines.md).

### CODE REVIEW

**Comment by promptws** [comment]
View a preview at https://prompt.ws/r/Azure/azure-cli/5848
This is an **experimental preview** for @microsoft users.

File: src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py [hunk 1]
```diff
@@ -419,13 +419,14 @@
...
+                         domains.
         - name: --methods
-          short-summary: List of HTTP methods allowed to be executed by the origin.
+          short-summary: A space-separated list of HTTP methods allowed to be executed by the origin.
         - name: --allowed-headers
-          short-summary: List of response headers allowed to be part of the cross-origin request.
+          short-summary: A space-separated list of response headers allowed to be part of the cross-origin request.
         - name: --exposed-headers
-          short-summary: List of response headers to expose to CORS clients.
+          short-summary: A space-separated list of response headers to expose to CORS clients.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: For consistency with the other uses in the CLI this should just say "Space-separated list of..."

Comment (troydai) [reply src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: ... -_- ok

Comment (sptramer) [reply src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: @tjprescott I'd say that @troydai's wording here is correct, and the other documentation needs to change. While this change can be temporary here, there should be a ticket for batch-editing the help.

Comment (tjprescott) [reply src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py]: It wouldn't just be instances of space-separated lists.  In many, many instances in the CLI we don't lead the help text with an article. It adds no clarity, so for the sake of brevity it's often (I'd argue usually) omitted.

In this specific case, I don't know of any uses of space-separated list that use an article.

**Pull request merged**